index.html

```diff
@@ -10,11 +10,72 @@
 </head>
 <body>
     <span class= "title"> Front-End Tools </span>
-    <div class="tech">
-      <img class="tech-img" src="assets/react_icon.svg">
-      <h4 class="text">ReactJS</h4>
-      <h4 class="text text-bold">Release: 2013</h4>
-      <span class="text">React is declarative, component-based JavaScript library for building user interfaces.</span>
+    <div class="tech-container">
+      <div class="tech">
+        <img class="tech-img" src="assets/react_icon.svg">
```

Comment:
Add `/` at the end of `<img>` tags `<img class="tech-img" src="assets/react_icon.svg"/>` that way